server/command/install.go

```diff
@@ -4,91 +4,53 @@
 package command
 
 import (
-	"fmt"
-
 	"github.com/pkg/errors"
 	"github.com/spf13/pflag"
 
 	"github.com/mattermost/mattermost-server/v5/model"
 
 	"github.com/mattermost/mattermost-plugin-apps/apps"
-	"github.com/mattermost/mattermost-plugin-apps/server/api/impl/proxy"
 	"github.com/mattermost/mattermost-plugin-apps/server/http/dialog"
 )
 
 func (s *service) executeInstall(params *params) (*model.CommandResponse, error) {
 	appID := ""
-	manifestURL := ""
 	appSecret := ""
-	force := false
 	fs := pflag.NewFlagSet("", pflag.ContinueOnError)
 	fs.StringVar(&appID, "app-id", "", "ID of the app")
-	fs.StringVar(&manifestURL, "url", "", "manifest URL")
```

Comment:
Should we keep the `--url` parameter to allow the installation with only one command, not two?